encoder-brave/src/main/java/zipkin2/reporter/otel/brave/SpanTranslator.java

```diff
@@ -172,6 +184,28 @@ private Span.Builder builderForSingleSpan(MutableSpan span, ResourceSpans.Builde
     return spanBuilder;
   }
 
+  void resolveAddressAndPort(MutableSpan span, Supplier<String> fallbackAddress, Supplier<Integer> fallbackPort, BiConsumer<String, Integer> addressAndPortConsumer) {
```

Comment:
one way to potentially reduce clutter is to use use the real values for fallback address and port instead of passing methods here. If you look at MutableSpan, it is simple field access, so cheaper and easier to read to use that. unless I'm missing something.

Reply:
Well, there's no particular reason to use Supplier here.

Addressed in https://github.com/openzipkin-contrib/zipkin-otel/pull/28/commits/70e673fe952ae00178b8234c1a96216b844cbd43